sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_sender.py

```diff
@@ -68,6 +68,8 @@ def _build_schedule_request(cls, schedule_time_utc, *messages):
             if not isinstance(message, Message):
                 raise ValueError("Scheduling batch messages only supports iterables containing Message Objects."
                                  " Received instead: {}".format(message.__class__.__name__))
+            if isinstance(message, (PeekMessage, ReceivedMessageBase)):
```

Comment:
Typical "pythonic" nonsense question, is there a way that we could do this without an explicit typecheck via a try/except?

If not then I won't sweat it but I have to ask :P

That said the potentially larger question here, wouldn't you need isSubClass for ReceivedMessageBase vs isInstance?

Reply:
Instead of class method on Message, I make it a private method on PeekMessage, so that we could that `except AttributeError` to control the flow which is more pythonic.

How does it look to you know?

Reply:
++ Very nice.